frontend/src/components/experimentRuns/ExperimentRunsListContent.js

```diff
@@ -80,14 +73,24 @@ const getTableColumns = (containersByID, runTypes, instruments) => [
     sorter: true,
     width: 180,
   },
-
+  {
+    title: "Launch",
+    dataIndex: "run_processing_launch_date",
+    sorter: true,
```

Comment:
Sorter doesn't work (maybe it does but I have to launches, each with same dates but different time and it does not reorder them)

Reply:
Fixed - I was missing the filter key for run_processing_launch_date